public/app/plugins/panel/geomap/GeomapTooltip.tsx

```diff
@@ -19,14 +19,14 @@ export const GeomapTooltip = ({ ttip, onClose, isOpen }: Props) => {
   const { dialogProps } = useDialog({}, ref);
 
   return (
-    <>
+    <Portal>
```

Comment:
any sense if we should add portal all the time, or only when the tooltip is in use?

If adding it conditionally is not worse, we should do that since it will have less *stuff* in the real DOM unless it is actually used

Reply:
You're right, updated!